automated install/uninstall.sh

```diff
@@ -198,7 +208,13 @@ else
   echo -e "  ${INFO} Be sure to confirm if any dependencies should not be removed"
 fi
 while true; do
-	read -rp "  ${QST} Do you wish to go through each dependency for removal? [Y/n] " yn
+  echo -e "${COL_YELLOW}  ${INFO} The following dependencies may have been added to the system by Pi-hole install:"
```

Comment:
`${COL_YELLOW}` should be placed after `${INFO}` and the line should be limited to 80 characters.

I propose: `echo -e "  ${INFO} ${COL_YELLOW}The following dependencies may have been added by the Pi-hole install:"`

Reply:
feel free to push that change on my behalf, please!